web-ui/src/main/resources/catalog/views/default/templates/recordView/featurecatalogue.html

```diff
@@ -1,7 +1,34 @@
-<div data-ng-repeat="featureType in mdView.current.record.featureTypes">
-  <h3 data-translate="">featureAttributeTable</h3>
+<div
+  class="panel panel-default gn-margin-top width-50"
+  data-ng-show="mdView.current.record.featureTypes.length > 0"
+>
+  <div class="panel-heading" data-gn-slide-toggle="true">
+    <span class="text-uppercase" data-translate="">tableOfContents</span>
+  </div>
+  <div class="panel-body">
+    <ul class="list-group">
+      <li
+        class="list-group-item"
+        data-ng-repeat="featureType in mdView.current.record.featureTypes"
+      >
+        <a
+          href
+          data-ng-click="scrollToSection('gn-featurecatalog-spy-target-' + featureType.code)"
```

Comment:
```suggestion
          data-ng-click="scrollToSection('gn-featurecatalog-spy-target-' + ((featureType.code + '-' + featureType.typeName) | md5))"
```

use a hash of code and name so that it is unique ? If code was empty, it was not working. With this it will be ok if code or name is defined. Require the next change too.